### PR TITLE
Updated default header size to big debug strings

### PR DESCRIPTION
**Pull request by uonick**
Hi! 

**Many** errors in `Laravel` have a **huge** number of rows and resize action (on hover) in div is **very** slow.

To speed up the development and ease the work of programmers, I suggest immediately making a window above!

### before

screen shot 2018-04-19 at 17 59 18

### after

screen shot 2018-04-19 at 17 59 32

### CODE REVIEW

File: src/Whoops/Resources/css/whoops.base.css [hunk 1]
```diff
@@ -38,7 +38,7 @@ header {
   box-sizing: border-box;
   background-color: #2a2a2a;
   padding: 35px 40px;
-  max-height: 180px;
+  max-height: 500px;
```
Comment (staabm) [inline src/Whoops/Resources/css/whoops.base.css]: Maybe this should be relative to the viewportheight with `vh` units?

Comment (uonick) [reply src/Whoops/Resources/css/whoops.base.css]: Yes, sure

Comment (uonick) [reply src/Whoops/Resources/css/whoops.base.css]: `vh` - does not give the necessary result...

**Comment by denis-sokolov** [comment]
This is a difficult one. Sometimes longer exception messages are more useful and sometimes they’re not useful, while a stacktrace is. For now I will not include this, but I’ll be on the lookout for how to solve this better. Thanks!

**Pull request closed without merging**